main.nf

```diff
@@ -298,15 +319,15 @@ workflow {
   main:
     if (params.wfmash == false) {
       edyeet(fasta)
-      seqwish(edyeet.out)
+      seqwish(fasta, edyeet.out)
     } else {
       wfmash(fasta)
-      seqwish(wfmash.out)
+      seqwish(fasta, wfmash.out)
```

Comment:
Did you add this because seqwish usually expects a FASTA and a PAF as input?

Reply:
Yes, the FASTA provides the sequences, while the PAF provides information on how they align with each other.